tests/tests/hafs_regional_atm_thompson_gfdlsf

```diff
@@ -0,0 +1,98 @@
+###############################################################################
+#
+#  HAFS regional atmosphere only
```

Comment:
@ChunxiZhang-NOAA, Please change this line into:
HAFS regional atmosphere only test with the Thompson MP and GFDL surface layer schemes
or something similar.

Reply:
Changed as you suggested